README.md

```diff
@@ -257,6 +257,10 @@ Setting up proper trust- and keystores has to be managed outside of the puppetdb
 
 If true, the module will attempt to connect to the database using the specified settings and fail if it is not able to do so. (defaults to true)
 
+####`database_path`
```

Comment:
Done, you'll want to git diff -w, since we still use the aligning arrows/equals style here and I had to line up a lot of stuff since database_embedded_path is long :-).